streaming/java/streaming-runtime/src/main/java/org/ray/streaming/runtime/serialization/Serializer.java

```diff
@@ -0,0 +1,9 @@
+package org.ray.streaming.runtime.serialization;
+
+public interface Serializer {
+
+  byte[] serialize(Object object);
+
+  Object deserialize(byte[] bytes);
```

Comment:
We  can return `T` as ray-runtime `Serializer` does. It helps on fail fast and avoid user of this method manually cast type.

This sort, including raw usage of generic class, among the code, annoyingly break developers' experience but they just pass compilation and right. Though, we can improve quite a bit indeed.

Reply:
Good suggestion, just fixed